src/hooks/sync-sites/use-sync-push.ts

```diff
@@ -225,7 +235,15 @@ export function useSyncPush( {
 			const file = new File( [ archiveContent ], 'loca-env-site-1.tar.gz', {
 				type: 'application/gzip',
 			} );
-			const formData = [ [ 'import', file ] ];
+
+			const formData = [];
+
+			formData.push( [ 'import', file ] );
+
+			if ( options?.optionsToSync ) {
+				formData.push( [ 'options', options.optionsToSync ] );
```

Comment:
I think sending an array won't work. The backend will override the latest value. We'll need to join the elements with `,`.